### PR TITLE
Editorial updates from Ivan

### PR DESCRIPTION
**Pull request by rhiaro**
Thanks to @iherman for these in #401

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/414.html" title="Last updated on Oct 25, 2020, 9:43 AM UTC (2049d55)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/414/c764c67...2049d55.html" title="Last updated on Oct 25, 2020, 9:43 AM UTC (2049d55)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -289,18 +287,18 @@ <h1>
...
     </p>
     <p>
-This specification does not require any particular technology or cryptography to
-underpin the generation, persistence, resolution or interpretation of DIDs.
-Rather, it defines: a) the generic syntax for all DIDs, and b) the generic
-requirements for performing the four basic CRUD operations (create, read,
-update, deactivate) on the metadata associated with a DID (called the DID
-document).
+This specification does not presuppose any particular technology or
+cryptography to underpin the generation, persistence, resolution or
```
Comment (iherman) [inline index.html:291]: Shouldn't this be "persupposes" ?

**Review by iherman — APPROVED** [review]

**Comment by iherman** [comment]
I have ticked off, in #401, the items that are covered by this PR.

File: index.html (hunk 1 above)
Comment (iherman) [reply index.html:291]: Oops, mistake on my part! Closing this.

File: index.html [hunk 2]
```diff
@@ -568,7 +565,7 @@ <h2>
 typically asserted by the control of a set of cryptographic keys used by
 software acting on behalf of the controller, though it may also be asserted via
 other mechanisms. Note that a DID may have more than one controller, and the
-controller(s) may include the <a>DID subject</a>.
+<a>DID subject</a> can be the DID controller.
```
Comment (brentzundel) [inline index.html:568]:
```suggestion
<a>DID subject</a> can be the DID controller, or one of them.
```

File: terms.html [hunk 3]
```diff
@@ -180,8 +182,9 @@
   <dd>
 A <a>DID</a> plus any additional syntactic component that conforms to  the
 definition in <a href="#did-url-syntax"></a>. This includes an optional <a>DID
-path</a>, optional <a>DID query</a> (and its leading <code>?</code> character),
-and optional <a>DID fragment</a> (and its leading <code>#</code> character).
+path</a> (and its leading <code>/</code> character), optional <a>DID query</a>
+(and its leading <code>?</code> character), and optional <a>DID fragment</a>
+(and its leading <code>#</code> character).
```
Comment (brentzundel) [inline terms.html:187]:
```suggestion
path</a> (with its leading <code>/</code> character), optional <a>DID query</a>
(with its leading <code>?</code> character), and optional <a>DID fragment</a>
(with its leading <code>#</code> character).
```

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**